openmls/src/extensions/capabilities_extension.rs

```diff
@@ -94,45 +94,50 @@ impl CapabilitiesExtension {
             },
         }
     }
+
     /// Get a reference to the list of versions in this extension.
     pub fn versions(&self) -> &[ProtocolVersion] {
         self.versions.as_slice()
     }
+
     /// Get a reference to the list of ciphersuites in this extension.
     pub fn ciphersuites(&self) -> &[Ciphersuite] {
         self.ciphersuites.as_slice()
     }
+
     /// Get a reference to the list of supported extensions.
     pub fn extensions(&self) -> &[ExtensionType] {
         self.extensions.as_slice()
     }
+
     /// Get a reference to the list of supported proposals.
     pub fn proposals(&self) -> &[ProposalType] {
         self.proposals.as_slice()
     }
-    /// Check if this [`CapabilitiesExtension`] supports all the capabilities
-    /// required by the given [`RequiredCapabilities`] extension. Returns
-    /// `true` if that is the case and `false` otherwise.
-    pub(crate) fn supports_required_capabilities(
-        &self,
-        required_capabilities: &RequiredCapabilitiesExtension,
-    ) -> bool {
-        // Check if all required extensions are supported.
-        if required_capabilities
-            .extensions()
-            .iter()
-            .any(|e| !self.extensions().contains(e))
-        {
-            return false;
-        }
-        // Check if all required proposals are supported.
-        if required_capabilities
-            .proposals()
-            .iter()
-            .any(|p| !self.proposals().contains(p))
-        {
-            return false;
-        }
-        true
-    }
+
+    // /// Check if this [`CapabilitiesExtension`] supports all the capabilities
```

Comment:
Unused now.

Reply:
The capabilities extension can be completely removed. These checks are on the `Capabiblities` struct in the `LeafNode` now.